lib/ts/controllers/s-modal.ts

```diff
@@ -97,6 +96,7 @@ namespace Stacks {
 
             if (toShow) {
                 this.bindDocumentEvents();
+                this.focusInsideModal();
```

Comment:
This should be safe to do on each show because the event has `once: true` set.

No changes necessary.